diff --unified --recursive --text bumblebee.orig/src/module.c bumblebee/src/module.c
--- bumblebee.orig/src/module.c	2020-08-12 16:07:40.327134382 +0000
+++ bumblebee/src/module.c	2020-08-12 16:08:01.953801276 +0000
@@ -96,7 +96,8 @@
     int retries = 30;
     bb_log(LOG_INFO, "Unloading %s driver\n", driver);
     char *mod_argv[] = {
-      "rmmod",
+      "modprobe",
+      "-r",
       driver,
       NULL
     };
